### PR TITLE
Feat: Expose html attrs for webpack output

### PR DESCRIPTION
**Pull request by vxna**
TIL styleguidist/mini-html-webpack-plugin#6 is not going to be implemented and I am myself in need of this feature so I've decided to implement this at template level as I think it's a more suitable place for it.

There is no breaking changes I am aware of but I am going to release this as 1.0.0 anyways.

/cc @sapegin thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,22 +55,19 @@ module.exports = {
...
-              rel: 'icon',
-              type: 'image/x-icon',
-              href: 'https://assets-cdn.github.com/favicon.ico'
             }
           ]
         },
         body: {
           raw: '<div id="root"></div>'
+        },
+        attrs: {
```
Comment (sapegin) [inline README.md:66]: I'm not really sure what HTML this will produce. Why is it an array?

**Review by sapegin — COMMENTED** [review]
I agree that this is a better place for a feature!

File: README.md (hunk 1 above)
Comment (vxna) [reply README.md:66]: From the tests, this producing valid HTML. Say by default `mini-html-webpack-plugin` prints webpack output script tags as `<script src="/build.js"></script>`, so this feature adds support for userland attributes like `defer` or [others](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script) and for the readme example output would be `<script async="" type="text/javascript" src="/build.js"></script>`.

I've maked this as an array for keeping [current api](https://github.com/vxna/mini-html-webpack-template/tree/feat/attrs#extended-options) consistent.

**Comment by sapegin** [comment]
And if there are more than one script? Will it apply the attributes to all of them? An array makes an impression that you can control each script tag separately.

Artem Sapegin, sapegin.me

**Comment by vxna** [comment]
Yes, it applies attributes for multiple files and this is covered by tests already. I can't imagine such use case where one would want to cherry-pick webpack script tags, they all could be only `async` or `defer` after all. 

For other needs we already have `header/body.script` api.

**Comment by sapegin** [comment]
Yup, that’s why array is misleading here.

Artem Sapegin, sapegin.me

**Comment by vxna** [comment]
Gotcha. I committed impoved behaviour.

**Comment by sapegin** [comment]
I think it's better now!

**Pull request merged**